src/PerformanceTests/Common/Scenarios/PublishOneOnOne/PublishOneOnOneRunner.V6.cs

```diff
@@ -0,0 +1,18 @@
+﻿#if Version6
+using NServiceBus;
+using System.Threading.Tasks;
+
+partial class PublishOneOnOneRunner
+{
+    public partial class Handler : IHandleMessages<Event>
+    {
+        public async Task Handle(Event message, IMessageHandlerContext ctx)
+        {
+            if (Shutdown) return;
+            await ctx.Publish(message).ConfigureAwait(false);
```

Comment:
How about `if (!Shutdown) await Bus.Publish(message)` so that it aligns with the v5 handler?